fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/cascades/TypeRepositoryTest.java

```diff
@@ -199,30 +217,6 @@ void attemptToCreateArrayConstructorValueWithDifferentChildrenTypesFails() {
         }
     }
 
-    @Test
-    void createArrayConstructorValueWorks() {
-        final Typed value = new AbstractArrayConstructorValue.ArrayFn().encapsulate(null, List.of(INT_1, INT_2));
```

Comment:
This test doesn't apply any more, because we removed the code around null elements in array.